src/interfaces/coral_web/src/components/Shared/Markdown/Markdown.tsx

```diff
@@ -52,7 +54,7 @@ export const getActiveMarkdownPlugins = (
 
   const rehypePlugins: PluggableList = [
     // remarkRaw is a plugin that allows raw HTML in markdown
-    rehypeRaw,
+    // rehypeRaw, // FIX: Disabled because it is causing issues with the rendering of the markdown
```

Comment:
just for future ref xD 
```suggestion
    // rehypeRaw, // FIX(@knajjars): Disabled because it is causing issues with the rendering of the markdown
```

Reply:
actually it's @tomtobac 🤣